docs/interactions/structured-messages/template.md

```diff
@@ -19,10 +19,11 @@ curl -X POST "https://[YOUR DOMAIN].api.digital.ringcentral.com/1.0/contents"
     "title": "Ringcentral, Inc.",
     "subtitle": "Cloud Business Communications.",
     "url": "github://github.com/ringcentral",
+    "target": "open",
     "url_fallback": "https://github.com/ringcentral",
     "url_text": "Github",
     "items": [
-      { "title": "Go to website", "type": "url", "url": "github://github.com/ringcentral", "url_fallback": "https://github.com/ringcentral" },
+      { "title": "Go to website", "type": "url", "url": "github://github.com/ringcentral", "url_fallback": "https://github.com/ringcentral", "target": "webview" },
```

Comment:
Not sure if we discussed this, but why are you putting the target attribute in the generic example?

Reply:
It hadn't been discussed, it's removed since 7e4180d